tests/scalers/aws-dynamodb-streams.test.ts

```diff
@@ -0,0 +1,209 @@
+import test from 'ava'
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import { CreateTableCommand, DescribeTableCommand, DeleteTableCommand, DynamoDBClient,} from '@aws-sdk/client-dynamodb'
+import { DynamoDBStreamsClient, DescribeStreamCommand } from '@aws-sdk/client-dynamodb-streams'
+import { createNamespace, sleep, waitForDeploymentReplicaCount } from './helpers'
+
+const awsRegion = 'ap-northeast-1'
```

Comment:
```suggestion
const awsRegion = 'eu-west-2'
```

please use this region, so we are in sync with other AWS tests we have.